### DISCUSSION TITLE
Custom control six Gang ZigBee Remote switch.

### DISCUSSION THREAD

**Discussion by cebik** [make-all/tuya-local#1046] [Show and tell]
I've got 6 buttons switch "six-Gang-ZigBee-Remote" { [aliexpress_link](https://aliexpress.com/i/1005003731082631.html) } 
Tuya local want me to choose between:
- linkoze_dual_button
- loratap_curtain_switch

So I created new custom device maybe someone need or author add to main project.

[loratap_six_gang_zigbee_remote_switch.yaml.txt](https://github.com/make-all/tuya-local/files/12486377/loratap_six_gang_zigbee_remote_switch.yaml.txt)

BTW Thanks authors for "Tuya for local"!

**Comment by make-all** [comment]
Superseded by PR #1067

**Comment by cebik** [comment]
**What version of ZigBee hub do you have?**
Please help!

I've tested 4 hub:
1) worked fine until I updated firmware to V1.19.2 :(
    https://allegro.pl/oferta/centralka-bramka-zigbee-3-0-tuya-smart-life-t710-13676835229
2) not all devices can be conected to HA.
    I see in DP map that json parent isn't set!
    https://aliexpress.com/item/1005002494242833.html
3) not working at all
    https://allegro.pl/oferta/centralka-bramka-wifi-tuya-smart-life-zigbee-3-0-12388141725
4) not working at all
    https://allegro.pl/oferta/centralka-bramka-zigbee-3-0-tuya-smart-life-lan-13767415513

I want to test/buy new:
https://aliexpress.com/item/1005005898660642.html
or
https://aliexpress.com/item/1005005589787174.html
